### ISSUE TITLE
Scale back Service Account Roles need for TESK Project in Openstack

### ISSUE THREAD

**Issue by susheel** [EMBL-EBI-TSI/TESK#38]
Migrated from Trello

**Comment by aniewielska** [comment]
Added new Roles, RoleBinding and Service Account. The configuration is the same for OpenStack/OpenShift and probably will also work for Minikube and Google. Would be nice to merge common parts of deployment to one single place.
Ingress additionally needed separate set of ServiceAccount/ClusterRole/CRBinding, which I copied from official project (https://github.com/nginxinc/kubernetes-ingress/tree/master/install/rbac).

To test it properly on one environment OpenStack/Openshift one should install all TESK objects (.yamls) from scratch and:
* create a supposedly successful task and check, if it completes in state Complete
* create a task with inputs/outputs or volumes (PVCs) and check, if it completes in state Complete. Also check, if PVC got properly deleted (we have an issue here, though)
* list created tasks in all view modes
* cancel a running task (we have some issues here, as well) and see, if the currently running executor gets deleted.
